### PR TITLE
add gtdb_validation_tk=0.1.6

### PR DESCRIPTION
**Pull request by aaronmussig**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by aaronmussig** [comment]
@dparks1134 @pchaumeil are you okay with being maintainers?

**Comment by aaronmussig** [comment]
Ready for review @conda-forge/help-python thanks!

**Comment by dparks1134** [comment]
I'd like to be a recipe maintainer!

File: recipes/gtdb_validation_tk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9a2c1a913b940dabbca584b62ffbae491a1202438aefd601ab36c16dc0693226
+
+build:
+  skip: true  # [win or py<36]
```
Comment (marcelotrevisani) [inline recipes/gtdb_validation_tk/meta.yaml:13]:
```suggestion
  noarch: python
```

File: recipes/gtdb_validation_tk/conda_build_config.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+channel_sources:
```
Comment (isuruf) [inline recipes/gtdb_validation_tk/conda_build_config.yaml:1]: This file is wrong. Should delete it

File: recipes/gtdb_validation_tk/LICENSE [hunk 3]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (isuruf) [inline recipes/gtdb_validation_tk/LICENSE:1]: Please open an issue upstream about packaging the license file in the sdist

**Pull request merged**